metaflow/plugins/kfp/kfp_get_workflow_uid.py

```diff
@@ -0,0 +1,29 @@
+def get_workflow_uid(
+    work_flow_name: str,
+) -> str:
+    """
+    The environment variables that this depends on:
+        POD_NAMESPACE
+    """
+    import os
+    import subprocess
+
+    command = [
+        "kubectl",
+        "get",
+        "workflow",
+        work_flow_name,
+        "--output",
+        "jsonpath='{.metadata.uid}'",
+    ]
+
+    namespace = os.environ.get("POD_NAMESPACE", default=None)
+    if namespace:
+        command.extend(["--namespace", str(namespace)])
+
+    print("command=", " ".join(command))
+
+    result = subprocess.run(command, stdout=subprocess.PIPE)
+    uid = result.stdout.decode("utf-8").strip("'")
+    print("uid=", uid)
```

Comment:
Leftover debug?

Reply:
It's intentional, it's useful info.  This step wouldn't be seen by the user.

![image](https://user-images.githubusercontent.com/4167032/118845905-09d24800-b881-11eb-84f7-6f789502d39e.png)

Reply:
Cool sounds good :)